### PR TITLE
Create stack command

### PR DESCRIPTION
**Pull request by chregu**
Further work on #15 - Works now as imagined. There was not much missing. Thanks a lot @jeanmonod

### CODE REVIEW

File: src/Command/StackCreateCommand.php [hunk 1]
```diff
@@ -63,9 +64,11 @@ protected function askForOperation($operations, $input, $output) {
                     continue;
                 }
                 $options[$optionName] = $this->askForOption($optionName, $property['type'], $input, $output);
+            } else {
+                $options[$optionName] = $this->askForOption($optionName, $property['type'], $input, $output);
```
Comment (jeanmonod) [inline src/Command/StackCreateCommand.php]: No need to duplicate this assignent line. Just move it outside of the second `if`....  My bad

Comment (chregu) [reply src/Command/StackCreateCommand.php]: fixed

File: src/Command/StackCreateCommand.php [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+            }
+            break;
+        }
+        return $data;
+    }
+
+    protected function displayResume($output){
+        $output->write("Creation of a new stack [<info>{$this->collectedData['name']}</info>]", true);
+        foreach($this->collectedData['operations'] as $name => $options){
+            $output->write(" * Operation [<info>$name</info>] with ".json_encode($options), true);
```
Comment (jeanmonod) [inline src/Command/StackCreateCommand.php]: Here, now, it should be:

    foreach($this->collectedData['operations'] as $name => $operation){
        $output->write(" * Operation [<info>$name</info>] with ".json_encode($operation->options), true);

Comment (chregu) [reply src/Command/StackCreateCommand.php]: fixed

**Comment by jeanmonod** [comment]
Thanks @chregu 👍 My first contribution ;)
I just took a look at https://github.com/rokka-io/rokka-client-php-cli/pull/16/files?w=1 and I saw a red line :( Sorry for that. So #17

**Pull request merged**